src/components/checkbox/Checkbox.scss

```diff
@@ -52,7 +58,7 @@
         }
 
         &:focus + span::before {
-            border: 2px solid $primary-color;
+            border: none;
```

Comment:
would this style always take priority because the specificity is higher (not 100% sure though) than the `.Checkbox--isFocused` selector?

Reply:
That is a very good point, and come to think of it, I do not think I have not checked that the focus styles are indeed there without the prop. Probably should have written a story for that. But since I think we'll go the route of overriding from EUA, I will revert these changes to styles.

Reply:
(Pending verification that I can override them properly from EUA :) )